app/main.js

```diff
@@ -332,16 +376,28 @@ function startMicrobreak () {
       }
     }
 
-    if (!(settings.get('fullscreen') && process.platform === 'win32')) {
+    if (settings.get('fullscreen') && process.platform === 'linux') {
+      windowOptions.width = displaysWidth(displayIdx)
+      windowOptions.height = displaysHeight(displayIdx)
+      windowOptions.x = displaysX(displayIdx, 0, true)
+      windowOptions.y = displaysY(displayIdx, 0, true)
+    } else if (!(settings.get('fullscreen') && process.platform === 'win32')) {
       windowOptions.x = displaysX(displayIdx)
       windowOptions.y = displaysY(displayIdx)
     }
 
+    console.log('monitors id', displayIdx)
+    console.log('monitors windowOptions', windowOptions)
     let microbreakWinLocal = new BrowserWindow(windowOptions)
     // microbreakWinLocal.webContents.openDevTools()
+    console.log('microbreakWinLocal.Bounds after creation', microbreakWinLocal.getBounds())
+    microbreakWinLocal.setSize(windowOptions.width, windowOptions.height)
```

Comment:
@dbankmann do we need this? it causes error in regular sized window as `windowOptions.height` does not exist

we are settings it on line 379 so I would say this is duplicated

Reply:
Does this work?
```suggestion
    microbreakWinLocal.setSize(widht = windowOptions.width, height = windowOptions.height)
```
I removed the call via keywords in the last commit, as it produced linting errors.

This line is duplicate indeed, but if I remember correctly this was the essential part of the multi-head fix. Somehow the constructor `BrowserWindow` doesn't get the settings right. I'm not at my multi-head machine atm, so cannot test.

Reply:
You mean multiple displays? Will try to test today. Electron sucks in this that some parameters seems to work only when called via method or vice-versa ;/ 

This is API, should be as you did in the end: https://www.electronjs.org/docs/api/browser-window#winsetsizewidth-height-animate 

```
win.setSize(width, height[, animate])

```

Anyway, the problem there is that windows.height does not exist as I am not in multiple display situation (line 484)

Reply:
I've added default width, height, so we don't get error, Kept `setSize` just in case